utils/kubeutil.py

```diff
@@ -93,5 +106,101 @@ def extract_kube_labels(self, pods_list, excluded_keys=None):
 
         return kube_labels
 
+    def extract_meta(self, pods_list, field_name):
```

Comment:
do we need this? It's nice and it's tested ( 🍪 ) but I don't see any usage of it.